servers/cu/src/domain/lib/evaluate.js

```diff
@@ -238,6 +240,12 @@ export function evaluateWith (env) {
                             ctx.stats.messages.error++
                           }
 
+                          /**
+                           * Increments gauges for total evaluation, total gas used
+                           */
+                          evaluationCounter.labels({ processId: ctx.id, cron: Boolean(cron), dryRun: Boolean(ctx.dryRun), error: Boolean(output.Error) }).inc()
+                          gasCounter.labels({ processId: ctx.id, cron: Boolean(cron), dryRun: Boolean(ctx.dryRun), error: Boolean(output.Error) }).inc(output.GasUsed ?? 0)
```

Comment:
`processId`, and `error` as labels will result in [high cardinatlity](https://last9.io/blog/how-to-manage-high-cardinality-metrics-in-prometheus/). Best labels are enumerated types.

Is there a way to use [exemplars](https://vbehar.medium.com/using-prometheus-exemplars-to-jump-from-metrics-to-traces-in-grafana-249e721d4192) for `process-id`  and then `type` label that can have a value of `cron-(error)? | scheduled-(error)? | dry-run-(error)?`? This way, label only has 6 possible values.